Library/Homebrew/bottle_checksum.rb

```diff
@@ -0,0 +1,19 @@
+# typed: true
+# frozen_string_literal: true
+
+require "checksum"
+
+# A formula's bottle block checksum.
+#
+# @api private
+class BottleChecksum < Checksum
+  extend Forwardable
+
+  attr_reader :tag, :cellar
+
+  def initialize(hexdigest, tag: nil, cellar: nil)
```

Comment:
```suggestion
  def initialize(hexdigest, tag:, cellar:)
```

as if these are both `nil`: this class should not have been called

Reply:
This class is gone now.